code/datums/shuttles.dm

```diff
@@ -39,8 +39,8 @@
 
 /datum/map_template/shuttle/emergency/bar
 	suffix = "bar"
-	name = "Emergency Escape Bar"
-	description = "Features include a bathroom, a quality lounge for the heads, and a \
+	name = "NTTV Charon (B)"
```

Comment:
Also personal nit, but... `Nano Trasen Transport Vessel` is kinda weird, imo it would be better as `Nanotrasen Transport Vessel (NTV)`